appinventor/blocklyeditor/src/blocks/text.js

```diff
@@ -497,3 +497,79 @@ Blockly.Blocks['text_is_string'] = {
   },
   typeblock: [{translatedName: Blockly.Msg.LANG_TEXT_TEXT_IS_STRING_TITLE}]
 };
+
+Blockly.Blocks['text_replace_mappings'] = {
+  // Replace all occurrences in mappings with their corresponding replacement
+  category: 'Text',
+  helpUrl: Blockly.Msg.LANG_TEXT_REPLACE_WITH_MAPPINGS_HELPURL,
```

Comment:
@ewpatton Is there any new functionality for auto-generating docs for blocks (like with the component markdown), or does it the same as before? (have to edit an .md/.html manually?)